README.md

```diff
@@ -1,60 +1,73 @@
-#### Outputs
-- The only output to this program will be if there is no locally stored word file. we will discuss how this word file is 
-obtained below, but if the file does not exists please have your code create a folder called 'word_data' and store the
-file as words.txt.
-- Be sure this file is not uploaded to the git repo, as we do not want to add anything to our repo besides code.
-### Hangman
-- This code will operate a game of hangman.
-- The code will first check to see if a file is stored locally and open it.
-- If not it will go up to http://norvig.com/ngrams/sowpods.txt and download the data.
-- You code should then parse this data into a list and store the data in the aforementioned location.
-- The program will then choose a word at random that is at least as long as the minimum length specified by the input 
-argument.
-- The code will loop until the word is either guessed correctly or the player has run of body parts.
-- The loop will display:
-    - The current state of the hangman
-    - The current state of the word, with '_' used as unguessed letters
-    - The letters available to choose
-    - Ask the user to choose a letter
-    - Notify the user if the letter is in the word, if it is not in the word, if the user has already guessed the
-    letter, if the input is more than one character, or if the chosen value is not a letter
-    - If the letter is wrong, or it has already been guessed, update the hangman, and restart the loop
-    - If the value is not a letter, or if the value is more than one character, do not penalize the user, but restart the loop
-    - If the letter is correct, update the guess and restart the loop 
-- A full hangman (a loss) will display as (use this as a guide):
-```
-|‾‾‾‾‾‾‾|
-|       O
-|      /|\
-|      / \
-|
-```
-- Upon the end of the game, notify the user what the word was and tell them whether they won or lost
\ No newline at end of file
+# hangman_ash
+### Game description: 
+This is a program runs the hangman game from command prompt. 
+Program chooses a random word from a text stored locally, if file not 
+found locally it downloads it from web into a specific directory. 
+Program then randomly selected a word which is greater in size then 
+the minimum value users enters to start the program.
+##### Then the game begins...
+#### To start the game:
+<ol>
+<li>First open command line
+<li>Go to the directory where program is located
+<li>Type in the following:
+
+```
+python hang_man.py -m (minimum length of word)
+```
+<li>Program then displays the mystery word status by displaying __ 
+
+```
+ _  _  _  _  _  _  _  _
+```
+<li>Program also gives option of character to choose from and update list
+
+```
+ _  _  T  T _ I _ _  
+Pick from these letters: A B C D E F G H ~~ J K L M O P Q R S ~~ U V W X Y Z
+Guess a letter:
+```
+<li>Program will output an error message in the input is not one character only
+
+```
+Please enter a string character
```

Comment:
The prompt here is unclear because a string implies multiple characters. Say something like "Please enter a character from the list"

Reply:
Changed wording for inputs and validation for better clarification